src/observe-tag.js

```diff
@@ -30,6 +40,7 @@ module.exports = tagFunction => {
 			// if an element had focus, copy over all the selection data (so we can copy it back later)
 			if (removedElementWithFocusData.index >= 0) {
 				const removedElementWithFocus = parentAndChildrenNodes[removedElementWithFocusData.index]
+				removedElementWithFocusData.tagName = removedElementWithFocus.tagName
```

Comment:
since we now search for the element we previously had focus on, we want to store the `tagName` (this way, if an input is surrounded by other different elements, we can still find it again and reapply focus)